src/app/components/graph/graph-viz.js

```diff
@@ -140,84 +142,113 @@ angular
-                    },
-                    show: true,
-                },
-                {
-                    id: 'docs',
-                    content: 'View documentation',
-                    selector: 'node',
-                    tooltipText: 'Jump to the documentation for this node',
-                    onClickFunction: function(event) {
-                        var target = event.target || event.cyTarget
-                        var unique_id = target.id();
-                        $state.go('dbt.' + target.data('resource_type'), {unique_id: unique_id});
-                        graph.hideGraph();
-                    },
-                    show: true,
-                },
-                {
-                    id: 'hide-before-here',
-                    content: 'Hide this and parents',
-                    selector: 'node',
-                    onClickFunction: function(event) {
-                        var target = event.target || event.cyTarget
-                        var unique_id = target.id();
-
-                        var node = project.node(unique_id);
-                        if (node) {
-                            var spec = selectorService.excludeNode(node, {parents: true});
-                            graph.updateGraph(spec)
-                        }
-                    },
-                    show: true,
-                },
-                {
-                    id: 'hide-after-here',
-                    content: 'Hide this and children',
-                    selector: 'node',
-                    onClickFunction: function(event) {
-                        var target = event.target || event.cyTarget
-                        var unique_id = target.id();
-
-                        var node = project.node(unique_id);
-                        if (node) {
-                            var spec = selectorService.excludeNode(node, {children: true});
-                            graph.updateGraph(spec)
-                        }
-                    },
-                    show: true,
-                },
-                {
-                    id: 'export-png',
-                    content: 'Export PNG',
-                    selector: 'node',
-                    coreAsWell: true,
-                    onClickFunction: function(event) {
-                        var options = {
-                            bg: '#005e7a'
-                        };
-                        var png64 = cy.png(options);
-                        var link = document.createElement('a');
-                        link.download = 'dbt-dag.png';  // sets the filename for the download
-                        link.href = png64;
-                        link.click();
-                    },
-                    show: true,
-                },
-            ],
-            menuItemClasses: ['graph-node-context-menu-item'],
-            contextMenuClasses: ['graph-node-context-menu']
+          menuItems: [
+            {
+              id: "jump",
```

Comment:
Why would you change all `'` to `"`? Not only does this confuse the diff (there are much less real changes than it looks like), but you also make the code inconsistent (above your change, everything still uses single-quotes).

Reply:
Hi @mkielar, completely agree on the issue with the quotation marks. I am not a JS expert and familiar with the DBT repo; if this is the only issue I propose reverting to the original single-quote convention this snippet of code affected, since the original creator didn't follow up on this.